language/vm/vm_runtime/src/interpreter.rs

```diff
@@ -0,0 +1,1204 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    code_cache::module_cache::ModuleCache,
+    counters::*,
+    data_cache::TransactionDataCache,
+    gas_meter::GasMeter,
+    identifier::{create_access_path, resource_storage_key},
+    loaded_data::{
+        function::{FunctionRef, FunctionReference},
+        loaded_module::LoadedModule,
+    },
+};
+use libra_logger::prelude::*;
+use libra_types::{
+    access_path::AccessPath,
+    account_address::AccountAddress,
+    account_config,
+    byte_array::ByteArray,
+    contract_event::ContractEvent,
+    event::EventKey,
+    identifier::{IdentStr, Identifier},
+    language_storage::ModuleId,
+    transaction::MAX_TRANSACTION_SIZE_IN_BYTES,
+    vm_error::{StatusCode, StatusType, VMStatus},
+    write_set::WriteSet,
+};
+#[cfg(any(test, feature = "instruction_synthesis"))]
+use std::collections::HashMap;
+use std::{collections::VecDeque, convert::TryFrom, marker::PhantomData};
+use vm::{
+    access::ModuleAccess,
+    errors::*,
+    file_format::{Bytecode, FunctionHandleIndex, LocalIndex, StructDefinitionIndex},
+    gas_schedule::{AbstractMemorySize, GasAlgebra, GasCarrier, GasUnits},
+    transaction_metadata::TransactionMetadata,
+    IndexKind,
+};
+use vm_runtime_types::{
+    loaded_data::struct_def::StructDef,
+    native_functions::dispatch::{dispatch_native_function, NativeReturnStatus},
+    value::{Locals, ReferenceValue, Struct, Value},
+};
+
+// Data to resolve basic account and transaction flow functions and structs
+lazy_static! {
+    /// The ModuleId for the Account module
+    pub static ref ACCOUNT_MODULE: ModuleId =
+        { ModuleId::new(account_config::core_code_address(), Identifier::new("LibraAccount").unwrap()) };
+    /// The ModuleId for the Event
+    pub static ref EVENT_MODULE: ModuleId =
+        { ModuleId::new(account_config::core_code_address(), Identifier::new("Event").unwrap()) };
+
+}
+
+// Names for special functions and structs
+lazy_static! {
+    static ref CREATE_ACCOUNT_NAME: Identifier = Identifier::new("make").unwrap();
+    static ref ACCOUNT_STRUCT_NAME: Identifier = Identifier::new("T").unwrap();
+    static ref EMIT_EVENT_NAME: Identifier = Identifier::new("write_to_event_store").unwrap();
+}
+
+/// `Interpreter` instances can execute MOVE functions.
```

Comment:
`Move`?

Reply:
Changed all `MOVE` in the file to `Move`